src/__stories__/menu-story.tsx

```diff
@@ -35,39 +45,42 @@ export const Default: StoryComponent = () => {
     return (
         <Stack space={16}>
             <SectionTitle>Menu</SectionTitle>
-            <Menu
-                width={400}
-                renderTarget={({ref, onPress, isMenuOpen}) => (
-                    <Touchable elementRef={ref} onPress={onPress}>
-                        <Inline space={16}>
-                            <IconKebabMenuLight />
-                            <Text3 regular>{isMenuOpen ? 'Close' : 'Open'}</Text3>
-                        </Inline>
-                    </Touchable>
-                )}
-                renderMenu={({ref, className, close}) => (
-                    <div ref={ref} className={className}>
-                        {options.map((option) => (
-                            <Box paddingX={16} paddingY={8} key={option.value}>
-                                <Checkbox
-                                    name={option.text}
-                                    onChange={() => {
-                                        if (option.value === 'option3') {
-                                            setTimeout(() => {
-                                                close();
-                                            }, 400);
-                                        }
-                                        setValues(option.value);
-                                    }}
-                                    checked={valuesState.includes(option.value)}
-                                >
-                                    {option.text}
-                                </Checkbox>
-                            </Box>
-                        ))}
-                    </div>
-                )}
-            />
+            <div style={{display: 'flex', justifyContent: 'center'}}>
```

Comment:
can you add the menu as a snippet in Playroom? 🙏

Reply:
i'm not sure of what you want, is this link enough?

https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA6A7APAWRigrqgBPgO4CWUALgBYC8wALAAwMC%2BB%2BATtrOwCoCG7AOYxytABTAOMAGYAafBBQAFTgGdV8kqqy4A8gAds%2BZgEp81AHz4xbQmh4QcYSnwBGAGxj4YngLbZyACUZWk5pZgVlNVVaRRUYdQjVcgBPT1pJUgpKeHwARiZjZgtbQnw0AEkUdxIUL1V9PjAYWjyANmLSsvKKyBQAaRhXNx0cABkSQUpyfAB6EpRu7vsYAA9yAGYpQRx3AQtgLVGDIwB%2BfCQQAGF3CFUYS-xcy5OUS%2BY0WZ41zYWl8tmVRqdT%2BZU%2BDicLg8MFBJlYiykKG4o3EkjC8jAe3UADk%2BP4Mbd7sYzJZrKU0FASAA3KTSUIyCKYvg4vEtYBMln%2BToI7rAADaXTKwEF3XIP2eIAM5BIinyl1kIrKVL47hwMAlEH00sUeXliuYCp5S2FRv%2BYvWEqlMsWACY9aalsrVeqLiBNdqUHaQIb-oQDYKALoAOl8fH0YjE7utJKsNgddgAQhBVvhGlBKShBAANVodVN8dO1QQATVoAA4IgBrGApWJa61Bp1q7m%2BsFXSgwMCV1zJxXdFCsuseoPm8jw1vdRTtviZtliGP4E0TpYkaTWKOKRsqtXmajUV0blAbS5mJfLpb3cg8Ej%2BRzkCMLs-npaYu4wecAbj7voN%2BEYDBML94x-b9ukvABlcg%2BDFGwQEvAA1bcEnlBR603JsYEA0Cimw5xO2rKBaGEchIOg99LgQpDVBQvkAxMINakxHBYFUSM0JQLdnThb9QWXYBDxHH5xwnT523wntVl4pZPiTSTBRMbijU%2BSkqVheF5lQEBmCAA

Reply:
almost! add this code to playroom/snippets.tsx